tests/Unit/NumericalAlgorithms/Interpolation/InterpolationTargetTestHelpers.hpp

```diff
@@ -0,0 +1,263 @@
+// Distributed under the MIT License.
```

Comment:
Maybe just name the file `TargetTestHelpers.hpp` since it's already in `Interpolation`?

Reply:
I think it's more natural to keep this filename for now (so that it is consistent with everything else in the directory), and go back to give it a cleaner name as part of fixing issue #1095. If we're agreed, I can add a note to that issue reminding us to also reorganize the tests and this helper as well.